LEGO1/mxomni.cpp

```diff
@@ -289,3 +289,9 @@ MxEventManager* EventManager()
 {
   return MxOmni::GetInstance()->GetEventManager();
 }
+
+// OFFSET: LEGO1 0x1003ef40
+void SetAppCursor(WPARAM p_wparam)
+{
+    PostMessageA(MxOmni::GetInstance()->GetWindowHandle(), 0x5400, p_wparam, 0);
```

Comment:
Maybe we should start introducing shared named constants / enums for those values, since it looka like it interacts with `ISLE`: https://github.com/isledecomp/isle/blob/master/ISLE/isleapp.cpp

Reply:
I agree, that would probably be best going forward. Is this something you would like to be addressed in this PR?